app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,111 @@
+$(document).ready(function() {
+  var flag = true;
+  var flag2 = true;
+  $(".upvote").on("click", function( event ) {
```

Comment:
No rush right now, but I think it would be pretty cool if you guys refactored this code on the click later on to use the `this` keyword. I would add it as an issue to your github repo.